libbpf-cargo/src/btf/btf.rs

```diff
@@ -577,18 +581,34 @@ impl<'a> Btf<'a> {
         }))
     }
 
-    fn load_struct(&self, t: &btf_type, extra: &'a [u8]) -> Result<BtfType<'a>> {
+    fn load_struct(&mut self, t: &btf_type, extra: &'a [u8]) -> Result<BtfType<'a>> {
+        let name = match self.get_btf_str(t.name_off as usize)? {
+            "" => {
+                let n = format!("{}{}", ANON_AGGREGATE_PREFIX, self.anon_agg_count);
+                self.anon_agg_count += 1;
+                n
```

Comment:
why start with zero index? I think __anon_1 is a better starting point and allows to make this a bit nicer:

```
self.anon_agg_count += 1;
format!("{}{}", ANON_AGGREGATE_PREFIX, self.anon_agg_count)
```

Reply:
+1.  I wanted to do this originally :)  Will address.